mypy/build.py

```diff
@@ -893,6 +897,23 @@ def read_plugins_snapshot(manager: BuildManager) -> Optional[Dict[str, str]]:
     return snapshot
 
 
+def read_quickstart_file(options: Options) -> Optional[Dict[str, Tuple[float, int, str]]]:
+    quickstart = None  # type: Optional[Dict[str, Tuple[float, int, str]]]
+    if options.quickstart_file:
+        # This is very "best effort". If the file is missing or malformed,
```

Comment:
I feel like we should at least warn on a bad file. That will also allow our tests to catch bugs.

Reply:
Hm, I actually agree: warn but continue.

Reply:
mypy isn't very good at warning, though. We can label something as a warning, but it will still cause an error exit code.

We could just do a print, but then that will end up in the log file for the daemon (which is maybe fine?).

Reply:
`sys.stderr.write("warning: Blah\n")` sounds acceptable, assuming that will be visible when we run tests?

Reply:
Yeah the daemon tests capture stderr so it should be fine.

Reply:
It can be made visible by having the test dump the log file